cli_tools/common/disk/inspect.go

```diff
@@ -70,13 +78,14 @@ func (inspector *defaultInspector) Inspect(reference string, inspectOS bool) (ir
 		return
 	}
 
-	// TODO: Mute the UEFI detection results for now. Restore it till we got the reason of b/168671324
-	// ir.HasEFIPartition, _ = strconv.ParseBool(inspector.wf.GetSerialConsoleOutputValue("has_efi_partition"))
```

Comment:
We first need to roll out https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1365 before rolling this one out, correct?